internal/http/apitester_test.go

```diff
@@ -91,10 +96,11 @@ func TestAPITester_GetEventResult(t *testing.T) {
 	defer ts.Close()
 
 	c := &APITester{
-		HTTPClient: ts.Client(),
-		URL:        ts.URL,
-		Username:   "dummy",
-		AccessKey:  "accesskey",
+		HTTPClient:         NewRetryableClient(TestClientTimedOut),
+		URL:                ts.URL,
+		Username:           "dummy",
+		AccessKey:          "accesskey",
+		RequestRateLimiter: rate.NewLimiter(rate.Every(1*time.Second), 5),
```

Comment:
So that we're not throttling the test.
```suggestion
		RequestRateLimiter: rate.NewLimiter(rate.Inf, 0),
```